Add python image (#67)
kyma-incubator/third-party-images: python/README.md

> # python

Reviewer: ```suggestion
# Python
```

> This image contains python-alpine image built from github sources. It was based on <https://hub.docker.com/_/python>.

Reviewer: ```suggestion
This folder contains the `python-alpine` image built from GitHub sources. It is based on `https://hub.docker.com/_/python`.
```
